examples/glfw/FlutterEmbedderGLFW.cc

```diff
@@ -0,0 +1,156 @@
+
+#include <assert.h>
+#include <embedder.h>
+#include <glfw3.h>
+#include <chrono>
+#include <iostream>
+
+// If your flutter app is drawing at the wrong size you may need to adjust your
+// pixel ratio.
+static const double kPixelRatio = 2.0;
```

Comment:
See this discussion on inferring the pixel ratio from GLFW calls. https://www.glfw.org/docs/latest/news.html#news_30_hidpi. You don't need to hardcode this.

Reply:
Done